csunplugged/topics/management/commands/_ProgrammingChallengesStructureLoader.py

```diff
@@ -65,7 +65,7 @@ def load(self):
 
             prog_reminders_translations = self.get_blank_translation_dictionary()
             # scratch currently unsupported
```

Comment:
This comment may be worth updating. This block-based implementation isn't exactly Scratch but it's pretty close!